azure-pipelines.yml

```diff
@@ -0,0 +1,36 @@
+schedules:
+- cron: "0 7 * * SUN"
+  displayName: Weekly Sunday
+  branches:
+    include:
+    - master
+  always: true
+
+resources:
+  pipelines:
+  - pipeline: rosidl_typesupport_opendds
+    source: rosidl_typesupport_opendds
+    trigger: 
+      branches:
+      - master
+
+jobs:
+- job: Linux
+  timeoutInMinutes: 90
+  pool:
+    vmImage: 'ubuntu-18.04'
+  container: adamsjoci/opendds_ros2_docker:latest
+  steps:
+  - script: |
+      printenv
+        echo hello
```

Comment:
Is the 'echo hello' really needed here?

Reply:
The entire build step isn't really needed but I'm using it for now for trouble shooting. Will mark for removal.